elixir/apps/domain/lib/domain/config/definitions.ex

```diff
@@ -584,8 +584,15 @@ defmodule Domain.Config.Definitions do
     default: %{},
     sensitive: true,
     dump: fn map ->
-      Dumper.keyword(map)
-      |> Keyword.update(:tls_options, nil, &Dumper.dump_ssl_opts/1)
+      keyword = Dumper.keyword(map)
```

Comment:
@Intuinewin can you please run Elixir formatter? See https://github.com/firezone/firezone/actions/runs/9237615119/job/25661580627?pr=5126

Reply:
My bad